index.test.js

```diff
@@ -0,0 +1,54 @@
+import { exportAllDeclaration } from '@babel/types';
+import { TestScheduler } from 'jest';
+import { add, addString, toNumber, filterOrange } from './index';
+
+test('add 1 + 2 to equal 3', () => {
+	expect(add(undefined, 2)).toBe(NaN);
+});
+
+//Task01
+/*
+・aとbが"a"、"b"の時"ab"となる
+・aが"a", bが1の時"a1"が返る
+・aが1、bが1の時-1が返る
+*/
+
+describe('addString', () => {
+	test('add a + b to equal ab', () => {
+		expect(addString('a', 'b')).toMatch('ab');
+	});
+
+	//toMatch toBe
+
+	test('add a + 1 to match a1', () => {
+		expect(addString('a', 1)).toBe('a1');
+	});
+
+	// if = false else return -1
+	test('add 1 + 1 return -1', () => {
+		expect(addString(1, 1)).toEqual(-1);
```

Comment:
please explain different between `toMatch` and `toBe` and `toEqual`

Reply:
Thank you for reviw！
There are three differences.

.toMatch = Check the character string with a regular expression. 

.toBe = Test for strict equivalence. Compare numbers and strings.
             ex)'1' === 1 /false    number === string

.toEqual = Compare the values ​​of objects. 
                 [aaaa, eeee]  {a:01 b:02}

If the result is a string, I modified it to .toMatch.
If the value changes, I modified it to .toBe.

[1a744fa](https://github.com/nyattsu72/handson-jest/pull/1/commits/1a744fa90752a3bd4e2adbc90a031928c596fe80)